src/lifecycle.js

```diff
@@ -34,6 +34,11 @@ var matchesSelector = function (element, selector) {
   return nativeMatchesSelector.call(element, selector);
 };
 
+// Edge has a bug where oldValue is sent as null instead of the true oldValue
+// when an element attribute is removed. Bug raised at
+// https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/7711883/
+const needsAttrCaching = /Edge/.test(navigator.userAgent);
```

Comment:
In case anyone is wondering, to feature check this would take creating a mutation observer for an element add an attribute, then remove the attribute and then check the old value. Since mutation observers are run at the end of microtasks it could make the check flaky if other component code on the page is executed before the check.